runtime/polkadot/src/lib.rs

```diff
@@ -872,150 +874,10 @@ impl pallet_proxy::Trait for Runtime {
 pub struct CustomOnRuntimeUpgrade;
 impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		use frame_support::storage::{StorageMap, IterableStorageMap};
-		use pallet_democracy::{VotingOf, Conviction, Voting::Direct, AccountVote::Standard};
-		// Cancel convictions for Referendum Zero (for removing Sudo - this is something we would
-		// have done anyway).
-		for (who, mut voting) in VotingOf::<Runtime>::iter() {
-			if let Direct { ref mut votes, .. } = voting {
-				if let Some((0, Standard { ref mut vote, .. })) = votes.first_mut() {
-					vote.conviction = Conviction::None
-				}
-			}
-			VotingOf::<Runtime>::insert(who, voting);
-		}
-
-		<Runtime as frame_system::Trait>::MaximumBlockWeight::get()
+		purchase::remove_pallet::<Runtime>()
```

Comment:
executed both here and from the main lib.rs - which should it be?

Reply:
Main `lib.rs`? Right now it is executed in the Polkadot runtime and the Westend runtime. Westend probably doesnt have much in there, but we could test this simple cleanup on there first.